packages/playwright-core/src/server/bidi/third_party/firefoxPrefs.ts

```diff
@@ -54,10 +54,23 @@ function defaultProfilePreferences(
     // console
     // https://bugzilla.mozilla.org/show_bug.cgi?id=1543115
     'browser.dom.window.dump.enabled': true,
+
+    // Make sure newtab weather doesn't hit the network to retrieve weather data.
+    'browser.newtabpage.activity-stream.discoverystream.region-weather-config': '',
+
+    // Make sure newtab wallpapers don't hit the network to retrieve wallpaper data.
+    'browser.newtabpage.activity-stream.newtabWallpapers.enabled': false,
+    'browser.newtabpage.activity-stream.newtabWallpapers.v2.enabled': false,
+
+    // Make sure Topsites doesn't hit the network to retrieve sponsored tiles.
+    'browser.newtabpage.activity-stream.showSponsoredTopSites': false,
```

Comment:
This looks like a whack-a-mole game. Do you know why the "kill switch" `'browser.newtabpage.enabled': false` below is not sufficient and we still have to disable individual options?